scalalib/src/mill/scalalib/GenIdeaImpl.scala

```diff
@@ -321,7 +324,9 @@ object GenIdeaImpl {
                         compileOutputPath: Path,
                         generatedSourceOutputPath: Path,
                         libNames: Strict.Agg[String],
-                        depNames: Strict.Agg[String]) = {
+                        depNames: Strict.Agg[String],
+                        isTest: Boolean
```

Comment:
This allows us to display test sources nicely in sources tree, 
![image](https://user-images.githubusercontent.com/1951244/38783093-4d42fbfc-4106-11e8-8961-3bb936242fd2.png)

and split them from ordinary sources in search